debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/StreamingSourceIT.java

```diff
@@ -578,10 +578,10 @@ private void inconsistentSchema(EventProcessingFailureHandlingMode mode) throws
         start(MySqlConnector.class, config, (success, message, error) -> exception.set(error));
 
         try (
-                final MySqlTestConnection db = MySqlTestConnection.forTestDatabase(DATABASE.getDatabaseName());
-                final JdbcConnection connection = db.connect();
-                final Connection jdbc = connection.connection();
-                final Statement statement = jdbc.createStatement()) {
+                MySqlTestConnection db = MySqlTestConnection.forTestDatabase(DATABASE.getDatabaseName());
```

Comment:
Why is final obsolete here?

Reply:
I was unsure too so I checked in the past https://docs.oracle.com/javase/specs/jls/se8/html/jls-14.html#jls-14.20.3

> A variable declared in a resource specification is implicitly declared final if it is not explicitly declared final.